### PR TITLE
Nataliia dieieva

### PR DESCRIPTION
**Pull request by NataliiaDieieva**

### CODE REVIEW

File: source/app/states/dieieva/dieieva.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div id="dieieva" class="container">
```
Comment (course-js) [inline source/app/states/dieieva/dieieva.html:1]: you have the id of your page
Its good solution

File: source/styles/less/pages/dieieva.less [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.tuna {
```
Comment (course-js) [inline source/styles/less/pages/dieieva.less:1]: Why you did not use this id to cutting css rules only for your page ?

Comment (NataliiaDieieva) [reply source/styles/less/pages/dieieva.less:1]: Forgot that we are using less instead of css

**Review by course-js — APPROVED** [review]
OK. I will improve it myself.

**Pull request merged**